src/pages/loggedInView.vue

```diff
@@ -120,9 +120,11 @@ onMounted(async () => {
       userStore.privateKey = await getPrivateKey(userStore.privateKey)
     }
     devLogger.log('[loggedInView] after keygen', userStore.privateKey)
-    await setRpcConfigs()
-    await getRpcConfig()
-    await getKeySpaceType()
+
+    await Promise.all([
+      setRpcConfigs().then(async () => await getRpcConfig()),
```

Comment:
`() => getRpcConfig()` would do the same and be less expensive than instantiating an async function and would save a few bytes

Reply:
@mmjee fixed.